README.md

```diff
@@ -27,7 +27,35 @@ E.g. this code adds such ability to every Content NodeType:
 
 ![inspector editor](editor-demo.png)
 
-### 2. Render the anchors in the NodeType rendering
+### 2. Make content nodes available in the link search dialog of the backend
+
+Normally, Neos offers only document nodes for selection in its link search dialog. To list the corresponding content 
+nodes as well, you must allow this in the link configuration and set nodeTypes to 
+`['Neos.Neos:Document', 'CodeQ.JumpMarkers:Mixin.SectionConfiguration']`, for example:
+
+```
+'YOUR.Site:Content.Text':
+  superTypes:
+    'Neos.Neos:Content': true
+  ui:
+    label: Text
+    icon: file-text
+    position: 200
+    inlineEditable: true
+  properties:
+    text:
+      options:
+        ui:
+          inline:
+            editorOptions:
+              linking:
+                nodeTypes: ['Neos.Neos:Document', 'CodeQ.JumpMarkers:Mixin.SectionConfiguration']
```

Comment:
There's a bug with `options` and I would add two proper examples here:
```
  properties:
    text:
      type: string
      defaultValue: ''
      ui:
        inlineEditable: true
        inline:
          editorOptions:
            linking:
              nodeTypes: ['Neos.Neos:Document', 'CodeQ.JumpMarkers:Mixin.SectionConfiguration']
    link:
      type: string
      ui:
        label: 'Link'
        reloadPageIfChanged: false
        inspector:
          group: general
          editor: 'Neos.Neos/Inspector/Editors/LinkEditor'
          editorOptions:
            linking:
              nodeTypes: ['Neos.Neos:Document', 'CodeQ.JumpMarkers:Mixin.SectionConfiguration']

Reply:
Done.